scripts/battlefields/Cloister_of_Flames/waking_the_beast.lua

```diff
@@ -0,0 +1,71 @@
+-----------------------------------
+-- Area: Cloister of Flames
+-- BCNM: Waking the Beast
+-----------------------------------
+local cloisterOfFlamesID = zones[xi.zone.CLOISTER_OF_FLAMES]
+-----------------------------------
+
+local content = BattlefieldQuest:new({
+    zoneId           = xi.zone.CLOISTER_OF_FLAMES,
+    battlefieldId    = xi.battlefield.id.WAKING_THE_BEAST_CLOISTER_OF_FLAMES,
+    canLoseExp       = false,
+    maxPlayers       = 18,
+    timeLimit        = utils.minutes(30),
+    index            = 2,
+    entryNpc         = 'FP_Entrance',
+    exitNpc          = 'Fire_Protocrystal',
+    requiredKeyItems = { xi.ki.RAINBOW_RESONATOR },
+
+    questArea = xi.questLog.OTHER_AREAS,
+    quest     = xi.quest.id.otherAreas.WAKING_THE_BEAST,
+})
+
+function content:onEventFinishWin(player, csid, option, npc)
```

Comment:
> Not sure if Waking the Beast was converted to interaction yet, but if so, this could most likely move there

Ah, yeah I think the quest is not implemented yet. I have IF code for the quest but was going to wait until all BCNMs are merged to make the PR with the quest itself.